src/sentry/coreapi.py

```diff
@@ -391,8 +393,16 @@ def should_filter(self, project, data, ip_address=None):
         # TODO(dcramer): read filters from options such as:
         # - ignore errors from spiders/bots
         # - ignore errors from legacy browsers
-        if ip_address and not is_valid_ip(ip_address, project):
-            return True
+
+        filter_types = {
+            'error_messages': data.get('sentry.interfaces.Message', {}).get('message', []),
```

Comment:
I would love to see these keys move to an enum. Handing around strings like this can be very error prone since typos are not as easily caught or noticed. Whereas an enum would likely trigger an AttributeError.

So something like:

```python
class FilterTypes(object):
    ERROR_MESSAGES = 'error_messages'
    RELEASES = 'releases'
    BLACKLISTED_IPS = 'blacklisted_ips'
```

Then work with `FilterType.RELEASES` as a constant and never use the strings again.